email-provider/src/main/java/life/qbic/infrastructure/email/EmailSubmissionException.java

```diff
@@ -0,0 +1,23 @@
+package life.qbic.infrastructure.email;
+
+import java.io.Serial;
+
+/**
+ * <b>Email Submission Exception</b>
+ *
+ * @since 1.0.0
+ */
+public class EmailSubmissionException extends RuntimeException {
```

Comment:
What is the difference between an EmailSubmissionException and a CommunicationException?
What do they indicate? Can you elaborate in the JavaDoc?